content/body/table.php

```diff
@@ -513,17 +513,16 @@ class="sr-only">Distances</span></th>
 
 <ol>
     <li>It is CSS only.</li>
-    <li>It is easily searchable within the browser ... no extra coding required!</li>
-    <li>The UI is easy to understand and really straightforward.</li>
-    <li>If there are links inside the table, and you are concerned about the amount of tabstops there are inside the table for keyboard users, you could use <a href="skip-link.php">skip links</a> around the table    so they can skip this table entirely.</li>
+    <li>It is easily searchable within the browser ... no extra coding is required!</li>
+    <li>The UI is easy to understand and straightforward.</li>
+    <li>If there are links inside the table, and you are concerned about the number of tab stops there are inside the table for keyboard users, you could use <a href="skip-link.php">skip links</a> around the table    so they can skip this table entirely.</li>
 </ol>
 
 <p>
     If you are not worried about large hits to a backend database for generating
     the data or a ridiculously huge HTTP payload, I would
     suggest considering the use of a table with a fixed table header.  It has a
-    way less complicated UI, and if marked up correctly, is totally
-    accessible via screen readers and keyboard.
+    way less complicated UI, and if marked up correctly, is accessible via screen readers and keyboard.
```

Comment:
I think the following is what was trying to be conveyed in the original text:

```suggestion
    way less complicated UI, and if marked up correctly, is still accessible via screen readers and keyboard.
```